junit-platform-commons/src/main/java/org/junit/platform/commons/util/PackageUtils.java

```diff
@@ -119,7 +119,7 @@ public static void assertPackageNameIsValid(String packageName) {
 	 * @param name the attribute name as a string
 	 * @return an {@code Optional} containing the attribute value; never
 	 * {@code null} but potentially empty
-	 * @throws PreconditionViolationException if the supplied type is
+	 * @throws org.junit.platform.commons.PreconditionViolationException if the supplied type is
```

Comment:
:x: nit: We either need to import the exception from the new place or re-wrap the lines here to have a max of 80 characters.

Reply:
Why doesn't `spotlessApply` take care of it?

Reply:
Anyway, I manually formatted the lines and did _not_ introduce an explicit `import` for a type only needed in Javadoc context.

Reply:
The Eclipse formatter does not format comments.